src/extension.ts

```diff
@@ -49,6 +50,12 @@ export function activate(context: ExtensionContext): void {
 
   // start the client and the server
   client.start();
+
+  const disposable = vscode.commands.registerCommand(
+    'extension.ansible.vault',
+    toggleEncrypt
+  );
+  context.subscriptions.push(disposable);
```

Comment:
plz move this to the beginning of the function because the rest of the client-side setup is done on line 21 so I'd keep this closeby